### ISSUE TITLE
Twitter tip actions should be tab-selectable, and accessible via enter key

### ISSUE THREAD

**Issue by ryanml** [brave/brave-browser#5791]
Currently, you are unable to tab focus twitter tip actions on tweets, you should be able to tab-select them, then hit 'Enter' to engage the tipping banner. The goal is to have this achievable without clicks, for accessibility purposes.

**Comment by btlechowski** [comment]
Verification passed on

Brave | 0.70.109 Chromium: 77.0.3865.90 (Official Build) beta (64-bit)
-- | --
Revision | 58c425ba843df2918d9d4b409331972646c393dd-refs/branch-heads/3865@{#830}
OS | Ubuntu 18.04 LTS

Verified test plan from the description


![image](https://user-images.githubusercontent.com/34715963/66358834-e0e39480-e974-11e9-8be5-0e32e00d274a.png)

![image](https://user-images.githubusercontent.com/34715963/66358821-d1fce200-e974-11e9-8174-6645ebabe2c4.png)
